src/javascripts/events/navigationEvents.js

```diff
@@ -0,0 +1,53 @@
+import dashboardView from '../components/pages/dashboardView';
+import getCrew from '../helpers/data/crewData';
+import getSpecies from '../helpers/data/crudSpecies';
+import { showCrew, emptyCrew } from '../components/pages/crew';
+import { showReadSpecies, noReadSpecies } from '../components/pages/species';
+import { emptyLogEntry, showLogEntry } from '../components/pages/logEntry';
+import getLogEntry from '../helpers/data/logEntryData';
+import getDestinations from '../helpers/data/destinationsData';
+import destinationsView from '../components/pages/destinationsView';
+
+const navigationEvents = (user) => {
+  $('#navbarLogo').on('click', () => {
+    dashboardView();
+  });
+
+  document.querySelector('#readCrew').addEventListener('click', () => {
+    getCrew(user).then((crewArray) => {
+      if (crewArray.length) {
+        showCrew(crewArray, user);
+      } else {
+        emptyCrew();
+      }
+    });
+  });
+
+  $('#readDestinations').on('click', () => {
+    getDestinations().then((destinationsArray) => {
+      destinationsView(destinationsArray);
+    });
+  });
+
+  document.querySelector('#readLogEntries').addEventListener('click', () => {
+    getLogEntry(user).then((logArray) => {
+      if (logArray.length) {
+        showLogEntry(logArray, user);
+      } else {
+        emptyLogEntry();
+      }
+    });
+  });
+
+  document.querySelector('#readSpecies').addEventListener('click', () => {
+    getSpecies(user).then((speciesArray) => {
+      if (speciesArray.length) {
+        showReadSpecies(speciesArray, user);
+      } else {
+        noReadSpecies();
+      }
```

Comment:
So when clicking on the navbar as a authenticated user, I can see buttons for my `update` and `delete` however when on the dashboard and I click `Go There!` those buttons are not being rendered and this is for all resources.

Reply:
Just pushed up a bug fix!